src/SonarQube.Client.RequestGenerator/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,56 @@
+﻿/*
```

Comment:
I think you should remove some of the assembly properties from here and use the ones from the common assembly info.